providers/azure/resources/azure.lr

```diff
@@ -194,6 +196,94 @@ azure.subscription.computeService.disk @defaults("name location properties.osTyp
   properties dict
 }
 
+// Azure Batch
+private azure.subscription.batchService {
+  // Subscription identifier
+  subscriptionId string
+  // Batch accounts in the subscription
+  accounts() []azure.subscription.batchService.account
+}
+
+// Azure Batch account
+azure.subscription.batchService.account @defaults("id name location") {
```

Comment:
I'm not that deep into MQL resources, but when `azure.subscription.batchService` is private, shouldn't the sub-resources not also be private?